mac-android-emulator.sh

```diff
@@ -79,7 +79,7 @@ build_apk() {
 
 # Launches an existing emulator or creates an emulator and launches it.
 launch_emulator() {
-    if [ -z "$(emulator -list-avds)" ]; then
+    if [ -z "$($ANDROID_HOME/emulator/emulator -list-avds)" ]; then
```

Comment:
We use this path below, but not here, and so this fails (as it can't find the emulator binary).